app/views/projects/members/_member.html.erb

```diff
@@ -70,6 +70,45 @@
       <% end %>
     </div>
   </td>
+  <td class="px-6 py-4">
+    <% if (current_user != member.user && allowed_to?(:update_member?, @namespace)) && !member_source.key?(:inherited_namespace_path) %>
```

Comment:
Same thing applies here as the comment above

Reply:
a10c35a94bb03411803d11118e64ef39bdfb6913